web_src/css/actions.css

```diff
@@ -44,7 +44,7 @@
 }
 
 .run-list-item-right {
-  flex: 0 0 15%;
+  flex: 0 0 120px;
```

Comment:
?

Reply:
Before and after, both are not ideal:
<img width="397" alt="Screenshot 2024-04-06 at 19 28 24" src="https://github.com/go-gitea/gitea/assets/115237/2132edc3-ec1a-4407-8a5f-7f702a5a084c">
<img width="401" alt="Screenshot 2024-04-06 at 19 27 54" src="https://github.com/go-gitea/gitea/assets/115237/9067f6e0-eb8f-4097-a511-5e80d772a954">

Reply:
Tweaked it so mobile view is similar to before but now it does not waste space on desktop width:

<img width="447" alt="Screenshot 2024-04-06 at 19 34 12" src="https://github.com/go-gitea/gitea/assets/115237/6eea9a29-68e8-40b9-a4c7-4e072cfcd152">